firstvoices/backend/models/sites.py

```diff
@@ -132,6 +133,14 @@ class Meta:
     # from fvdialect:contact_email
     contact_email = models.EmailField(null=True, blank=True)
 
+    homepage = models.OneToOneField(
+        SiteWidgetList,
+        null=True,
+        blank=True,
```

Comment:
I think for this one we don't need blank=True, since it isn't a string field

Reply:
Don't we need blank=True here to allow a site to be created with no homepage widgets set?